### DISCUSSION TITLE
Conditional automation (webhooks) [Jun 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#2427] [Discord Threads]
> 🙋‍♂️ **jschwalbe**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### You can do it by introducing conditional triggers to `After Insert` event in webhook. Feel free to connect if you need more information.  
![image0](https://cdn.discordapp.com/attachments/983196903997075557/983242399507681320/Screenshot_2022-06-06_at_11.02.02_AM.png)  
#  
> 👨‍🦲 **dstala**  
#### You can find details here   
#### https://docs.nocodb.com/developer-resources/webhooks   
####    
#### images used in documentation is old. we will correct it up.  
#  
> 🙋‍♂️ **jschwalbe**  
#### Thank you -- not entirely sure I'm even asking this question correctly, but would this allow me to *redirect* someone to either more questions or an entirely different webpage on a different domain?  
#  
> 👨‍🦲 **dstala**  
#### this would post body that you have configured to the URL (different webpage on a different domain). you can select what data from the record that was just inserted to be sent.  
#  
> 🙋‍♂️ **jschwalbe**  
#### Thank you  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/983196903997075557)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
